src/styles/blocks/card.css

```
@@ -0,0 +1,74 @@
+.card {
+  width: 200px;
+  padding: 16px;
+  border: 1px solid #f3f3f3;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+}
+
+.card__main {
+  display: flex;
+  flex-direction: column;
+}
+
+.section1__image {
+  width: 160px;
+  height: 134px;
+  margin: 16px 3px 40px;
+}
+
+.section1__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin-bottom: 4px;
+}
+
+.section2__code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin-bottom: 16px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.section4__text {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.section4__number {
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__button {
+  width: 166px;
+  height: 40px;
+  background-color: #00acdc;
+  border-radius: 5px;
+  border: transparent;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  color: #fff;
+  text-transform: uppercase;
+}
```

better to split groups of modifiers by empty line
```suggestion
.card__button {
  width: 166px;
  height: 40px;
  
  background-color: #00acdc;
  border-radius: 5px;
  border: transparent;
  
  font-weight: 700;
  font-size: 14px;
  line-height: 16px;
  color: #fff;
  text-transform: uppercase;
}
```